### ISSUE TITLE
Android java.lang.NoClassDefFoundError: FakerService$$ExternalSyntheticLambda1

### ISSUE THREAD

**Issue by vad99lord** [serpro69/kotlin-faker#90]
I'm getting NoClassDefFoundError when creating Faker instance on android API 22. 
```
java.lang.NoClassDefFoundError: io.github.serpro69.kfaker.FakerService$$ExternalSyntheticLambda1
        at io.github.serpro69.kfaker.FakerService.load(FakerService.kt:67)
        at io.github.serpro69.kfaker.FakerService.<init>(FakerService.kt:32)
        at io.github.serpro69.kfaker.Faker.<init>(Faker.kt:14)
        at io.github.serpro69.kfaker.Faker.<init>(Faker.kt:13)
```
The issue seems to be connected with calling `defaultValues.merge()` in `private fun load(locale: String): Dictionary` in the following place:
```
readCategory(it, "en").entries.forEach { category ->
                if (defaultValues.containsKey(category.key)) {
                    defaultValues.merge(category.key, category.value) { t, u -> t.plus(u) }
                } else defaultValues[category.key] = category.value
            }
```
I can see that it's not available in Kotlin API and Java API is used. According to android docs [merge()](https://developer.android.com/reference/java/util/Map#merge(K,%20V,%20java.util.function.BiFunction%3C?%20super%20V,%20?%20super%20V,%20?%20extends%20V%3E)) is available since API 24 and  I have no issues running the code on API 30.

I don't know if you are going to handle this android-specific case but i think it could be possible to update merging using Kotlin API only similar to [this gist](https://gist.github.com/cy6erGn0m/97ecdc7191364572a94a).

**Comment by serpro69** [comment]
Hi @vad99lord ,

I have no issues supporting android-specific cases, but reproducing issues on android platform is harder for me because I don't work with android apps.
Would you be willing to make a PR with the fix? If not, I can also add a fix, but then it could potentially involve some back-and-forth to test if it's actually working.

**Comment by serpro69** [comment]
@vad99lord , I've removed usages of `merge` function from `FakerService` and this should be available in the latest `1.8.0-snapshot` version.
Please test and let me know if it's working, or re-open the issue otherwise.

**Comment by vad99lord** [comment]
Hello, @serpro69, sorry for the slow reply. The latest snaphot is working for me on both emulator and physical device with no problems. Thanks a lot for providing a quick fix to this issue.
